src/types.ts

```diff
@@ -65,6 +65,10 @@ export type BenchmarkOpts = {
   setupFiles?: string[];
   /** Trigger GC cleanup every test to have consistent memory usage */
   triggerGC?: boolean;
+  /** The algorithm to detect the convergence to stop the benchmark function runs */
+  convergence?: "linear" | "cv";
```

Comment:
what does cv mean? maybe include a brief description of each in the docstring

Reply:
Added the detailed comments.